moda-contracts/script/utils/DeployedContracts.sol

```diff
@@ -8,55 +8,16 @@ import {console2} from "forge-std/Script.sol";
 library DeployedContracts {
     Vm internal constant vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
 
-    function getModaRegistryAddress(uint256 chainId) public view returns (address) {
+    function get(string calldata scriptFileName, uint256 chainId) external view returns (address) {
         string memory filePath = string(
             abi.encodePacked(
-                "./broadcast/DeployModaRegistry.s.sol/", Strings.toString(chainId), "/run-latest.json"
+                "./broadcast/", scriptFileName, "/", Strings.toString(chainId), "/run-latest.json"
             )
         );
         string memory jsonContent = vm.readFile(filePath);
         string memory jsonPath = "$.transactions[0].contractAddress";
-        bytes memory modaRegistryBytes = vm.parseJson(jsonContent, jsonPath);
-        address modaRegistry = abi.decode(modaRegistryBytes, (address));
-        return modaRegistry;
-    }
-
-    function getManagementAddress(uint256 chainId) public view returns (address) {
-        string memory filePath = string(
-            abi.encodePacked(
-                "./broadcast/DeployManagement.s.sol/", Strings.toString(chainId), "/run-latest.json"
-            )
-        );
-        string memory jsonContent = vm.readFile(filePath);
-        string memory jsonPath = "$.transactions[0].contractAddress";
-        bytes memory managementBytes = vm.parseJson(jsonContent, jsonPath);
-        address management = abi.decode(managementBytes, (address));
-        return management;
-    }
+        bytes memory profileBytes = vm.parseJson(jsonContent, jsonPath);
```

Comment:
This can just be named `contractBytes ` or something else generic now

Reply:
ooh, whoops!